src/test/java/com/easypost/Fixture.java

```diff
@@ -0,0 +1,175 @@
+package com.easypost;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Fixture {
+
+    public static int pageSize() {
+        return 5;
+    }
+
+    public static String uspsCarrierAccountID() {
+        return "ca_8dc116debcdb49b5a66a2ddee4612600";
+    }
+
+    public static String childUserID() {
+        return "user_3878404c0bdd4321952d4cae45c1d184";
+    }
+
+    public static String usps() {
+        return "USPS";
+    }
+
+    public static String uspsService() {
+        return "First";
+    }
+
+    public static String reportStartDate() {
+        return "2022-02-01";
```

Comment:
Once you have created a few records, you'll most likely need these to be todays date. The Java user probably doesn't have data for this date range.